### DISCUSSION TITLE
how to show small context window around a match within a line

### DISCUSSION THREAD

**Discussion by IngenioCraft** [BurntSushi/ripgrep#2319] [Q&A] [answered]
I had a really hard time figuring this out, so wanted to share it here for others.  I'm new here... so this may only be of value to others starting out.
Had a very long .json file that did not have any line breaks, and wanted to pull some context around a search result by returning characters before and after my string.  If you have this challenge, try this format.  The following will pull 5 characters before and after the string:

rg -o ".{0,5}string_here.{0,5}" C:\Users\your_folder_or_file_etc

**Comment by BurntSushi** [comment]
Thank you for posting this!

**Comment by BurntSushi** [comment, marked as answer]
Just switching this to a Q&A and posting your answer (trying to keep things organized):

```
rg -o '.{0,5}string_here.{0,5}' path/to/file
```
